### PR TITLE
Feature: handle superfuild flow

### PR DESCRIPTION
**Pull request by 1yam**
The aleph instance create command now creates or updates the flow on the Avalanche mainnet.

The aleph instance delete command now deletes or updates the flow on the Avalanche mainnet.

## Self proofreading checklist

- [x] The new code clear, easy to read and well commented.
- [x] New code does not duplicate the functions of builtin or popular libraries.
- [ ] An LLM was used to review the new code and look for simplifications.
- [ ] New classes and functions contain docstrings explaining what they provide.
- [x] All new code is covered by relevant tests.
- [ ] Documentation has been updated regarding these changes.

## Changes
`aleph instance create` is now creating (or update if already exist) a flow when creating payg instance

`aleph instance delete` is now deleting (or update if already exist) a flow when creating payg instance

## How to test
```aleph instance create``` 
> choose superfluid & qemu

## Notes

If testing on node who is not with the new RC need to comment this line
in the create instance
```             requirements=HostRequirements(node=NodeRequirements(node_hash=crn.hash)) if crn else None,
```

This PR replace : [Pr](https://github.com/aleph-im/aleph-client/pull/240)

### CODE REVIEW

File: src/aleph_client/commands/instance/superfluid.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+logger = logging.getLogger(__name__)
+
+
+def from_wei(wei_value: Decimal) -> Decimal:
+    """Converts the given wei value to ether."""
+    return wei_value / Decimal(10**18)
+
+
+async def handle_flow(account: ETHAccount, chain: Chain, receiver: str, flow: Decimal):
```
Comment (hoh) [inline src/aleph_client/commands/instance/superfluid.py:21]: Can this be renamed `update_flow` and unified with `handle_flow_reduction` ?

File: src/aleph_client/commands/utils.py [hunk 2]
```diff
@@ -208,3 +213,18 @@ def has_nested_attr(obj, attr_chain) -> bool:
             return False
         obj = getattr(obj, attr)
     return True
+
+
+async def wait_for_processing(session, item_hash, account: ETHAccount, receiver: str):
```
Comment (hoh) [inline src/aleph_client/commands/utils.py:218]: Can you add type hints here ?

File: src/aleph_client/commands/instance/superfluid.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    flow_info: Web3FlowInfo = await account.get_flow(receiver)
+
+    if not flow_info["flowRate"]:
+        return await account.create_flow(receiver, flow)
+    else:
+        current_flow_rate_wei = Decimal(flow_info["flowRate"])
+
+        additional_flow_rate_wei = to_wei(flow, "ether")
+
+        new_flow_rate_wei = current_flow_rate_wei + additional_flow_rate_wei
```
Comment (hoh) [inline src/aleph_client/commands/instance/superfluid.py:47]: Can you add type annotations on this ?

**Comment by philogicae** [comment]
Overwritten by https://github.com/aleph-im/aleph-client/pull/258

**Pull request closed without merging**